### PR TITLE
[TACHYON-674] Update tiered storage documentation

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

File: docs/Tiered-Storage-on-Tachyon.md [hunk 1]
```diff
@@ -1,84 +1,187 @@
...
+
+## Reading Data
+
+Reading a data block with tiered storage is similar to standard Tachyon. Tachyon will simply read
+the block from where it is already stored. If Tachyon is configured with multiple tiers, then the
+block will not be necessarily read from the MEM tier, since it could have been moved to a lower tier
+transparently.
+
+Since the movement of blocks is transparent to the user (via allocators and evictors), Tachyon
+provides client APIs to *force* a block to be moved to the top tier. These APIs are:
```
Comment (calvinjia) [inline docs/Tiered-Storage-on-Tachyon.md]: Maybe add something about pinlist here?

**Comment by calvinjia** [comment]
Looks good overall, thanks for updating the docs!

File: docs/Tiered-Storage-on-Tachyon.md (hunk 1 above)
Comment (gpang) [inline docs/Tiered-Storage-on-Tachyon.md]: Added documentation about pinning files.

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
@calvinjia , please review this PR

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**